module/metrics/rest_api.go

```diff
@@ -6,27 +6,24 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus"
 
+	"github.com/onflow/flow-go/module"
+
 	httpmetrics "github.com/slok/go-http-metrics/metrics"
 	metricsProm "github.com/slok/go-http-metrics/metrics/prometheus"
 )
 
-// Example recorder taken from:
-// https://github.com/slok/go-http-metrics/blob/master/metrics/prometheus/prometheus.go
-type RestCollector interface {
-	httpmetrics.Recorder
-	AddTotalRequests(ctx context.Context, service string, id string)
-}
-
-type recorder struct {
+type RecorderCollector struct {
```

Comment:
Let's call this `RestCollector`. the previous "recorder" terminology is a bit confusing